packages/ipfs-daemon/package.json

```diff
@@ -44,6 +44,7 @@
   "dependencies": {
     "@ceramicnetwork/common": "^2.36.0-rc.0",
     "@ceramicnetwork/ipfs-topology": "^2.30.0-rc.0",
+    "dockerode": "^3.3.5",
```

Comment:
Now that we have the ceramic-one binary included, we don't need the Docker approach anymore, right? It can all be switched to launching the ceramic-one process?

Reply:
The docker approach provides a way to do an isolated embedded ceramic. If we don't think we need that functionality, can remove it.